src/einsteinpy/symbolic/predef/__init__.py

```diff
@@ -0,0 +1 @@
+from .de_sitter import AntiDeSitter, AntiDeSitterStatic, DeSitter
```

Comment:
Can we have something else than `predef` ? Any other ideas on a better word?  :confused:

Reply:
maybe `predefined` ? 
Anyways, `predef` seemed cute!

Reply:
LOL

Reply:
Any synonym? That sounds cool?

Reply:
no suitable synonym available. Search google!
going with `predefined`

Reply:
Resolved with commit 73830ca